server/schema.js

```diff
@@ -16,19 +13,14 @@ const query = new GraphQLObjectType({
     recentPost: {
       type: BlogPost,
       resolve() {
-        const coll = Collections.posts.rawCollection();
-        return sleep(200)
```

Comment:
Shall we keep these `sleep()` calls. They help us to show the latency compensation part. 

Use `Meteor._sleepFor(200)`